pkg/cmd/hgctl/dashboard.go

```diff
@@ -156,18 +165,26 @@ func consoleDashCmd() *cobra.Command {
   hgctl dash console
   hgctl d console`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if docker {
+				return accessDocker(cmd)
+			}
 			client, err := kubernetes.NewCLIClient(options.DefaultConfigFlags.ToRawKubeConfigLoader())
 			if err != nil {
-				return fmt.Errorf("build CLI client fail: %w", err)
+				fmt.Printf("build kubernetes CLI client fail: %v\n", err)
+				fmt.Println("try to access docker container")
```

Comment:
miss comma between two statements, by the way, it will be better to merge two fmt.printf to one.

Reply:
![image](https://github.com/alibaba/higress/assets/88550136/514f7620-463c-4295-b94e-14645c03dcad)
At present, the printing information is like this. Do you need to add a comma in the middle?

Reply:
it's ok, no need to add